app.js

```diff
@@ -0,0 +1,123 @@
+class ToDoList {
+    constructor() {
+        this.toDoTasks = [
+            // {
+            //     id: 1,
+            //     task: 'kupic 4 bulki',
+            //     completed: false,
+            // },
+            // {
+            //     id: 2,
+            //     task: 'przedluzyc abonament',
+            //     completed: false,
+            // },11`1
+            // {
+            //     id: 3,
+            //     task: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quasi amet incidunt, mollitia vero doloremque quisquam corporis veritatis eum dolore commodi esse iste ratione nemo. Cumque ad eaque quo. Sapiente, perspiciatis?',
+            //     completed: true,
+            // }
+        ];
+
+        document.getElementById('addTaskBtn').addEventListener('click', this.addTask);
+        this.inputAddTask = document.getElementById('addTaskInput');
+        this.taskSection = document.querySelector('.toDoTasks');
+
+        this.render();
+    }
+
+    addTask = (e, task = this.inputAddTask.value, id = Date.now()) => {
+        e.preventDefault();
+
+        if (task === '') return alert('cos chcialbys zapamietac, nie puste znaki...');
+        let taskItem = {
+            id: id,
+            task: task,
+            completed: false,
+        }
+
+        this.toDoTasks.push(taskItem);
+        this.inputAddTask.value = '';
+        this.render();
+    }
+
+    removeTask = (e, taskList = this.toDoTasks) => {
+        const idOfTaskToDelete = e.target.parentNode.dataset.key;
+
+        for (let i = 0; i < taskList.length; i++) {
+            if (taskList[i].id == idOfTaskToDelete) {
+                taskList.splice(i, 1);
+                break;
+            }
+        }
+
+        this.render();
+    }
+
+    changeStatus = (e, taskList = this.toDoTasks) => {
+        const idOfTaskToChange = e.target.parentNode.dataset.key;
+
+        for (let i = 0; i < taskList.length; i++) {
+            if (taskList[i].id == idOfTaskToChange) {
+                taskList[i].completed = e.target.checked;
+                break;
+            }
+        }
+
+        this.render();
+    }
+
+    clearView = () => {
+        const allTasks = document.querySelectorAll('.task');
+        allTasks.forEach(el => {
+            el.remove();
+        })
+
+    }
+
+    render = (taskList = this.toDoTasks) => {
```

Comment:
Ustawianie domyślnej wartości argumentu funkcji, która przy wywoływaniu nie ma żadnego przekazania elementu nie ma zbytnio sensu. Coś takiego stosujesz wtedy gdy dopuszczasz sytuację, że przekazany argument np jest `undefined`. W tym przypadku `render` wywołujesz w constructorze nieprzekazując nic. 
Także już lepiej zrobić wewnątrz `render`: `const taskList = this.toDoTasks` niż ustawiać domyślną wartość. 
Druga opcja to po prostu przekazanie do funkcji `render` listy i wtedy wywalenie domyślnej wartości.